week3/homework/src/actions/markAsDone.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+function markAsDone(todo, request, response) {
+  const id = request.params.id;
+  return todo.updateIn(id, true)
```

Comment:
Since `markAsDone()` and `markAsNotDone()` are exactly the same except for this `true` here and false in `markAsNotDone()`; this should be one function for example `toggleStatus(todo, status, {params}, response)`